tests/test_forms.py

```diff
@@ -0,0 +1,84 @@
+from django.test import TestCase
+
+from taxi.forms import (
+    DriverCreationForm,
+    CarForm,
+    ManufacturerSearchForm)
+from taxi.models import (
+    Driver,
+    Car,
+    Manufacturer)
+
+
+class DriverCreationFormTest(TestCase):
+    def test_driver_creation_form_valid(self):
+        form_data = {
+            "username": "testuser",
+            "password1": "testpassword123",
+            "password2": "testpassword123",
+            "license_number": "ABC12345",
+            "first_name": "John",
+            "last_name": "Doe",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_driver_creation_form_invalid_license(self):
+        form_data = {
+            "username": "testuser",
+            "password1": "testpassword123",
+            "password2": "testpassword123",
+            "license_number": "ab123",
+            "first_name": "John",
+            "last_name": "Doe",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertFalse(form.is_valid())
+        self.assertIn("license_number",
+                      form.errors)
+
+
+class CarFormTest(TestCase):
+    def setUp(self):
+        self.manufacturer = Manufacturer.objects.create(
+            name="Toyota", country="Japan")
+
+        self.driver = Driver.objects.create(
+            username="driver1",
+            password="password123",
```

Comment:
The password should be set using the `set_password` method to ensure it is hashed correctly. Consider using `self.driver.set_password('password123')` after creating the driver instance.